public/index.html

```diff
@@ -40,6 +40,197 @@
           <h1>Web Fundamentals: Web Systems</h1>
           <h2>Four Athens</h2>
         </section>
+          <section>
+              <div align="center">
```

Comment:
It's not a requirement, but adding `class="no-border"` to the images will supersede the need for the div here.